_docs/advanced-features.md

```diff
@@ -115,23 +115,25 @@ The latter is `~/.dockstore/cache/` by default and can be set to any directory l
 
 By default, Dockstore will attempt to download files up to three times. Control this with the `file-provision-retries` parameter inside `~/.dockstore/config`.
 
-## Running CWL-runner with extra tags
+## Running cwltool with extra tags
```

Comment:
I would say flags instead of tags, even though it was already like this before. Also, see my other comment; I think CWL-runner might be correct, although maybe it should be rephrased in such a way that it's not confused with `cwlrunner`.